web-stories.php

```diff
@@ -42,6 +42,7 @@
 }
 
 define( 'WEBSTORIES_VERSION', '1.0.0-alpha+e601d64' );
+define( 'WEBSTORIES_DB_VERSION', '1.0.0' );
```

Comment:
Should we just use incremental integers? Makes things a bit easier.

Reply:
This is much more human readable. We are using `version_compare` which makes well with 3 level version numbers.

Reply:
But do we need 3 level version numbers? 🙂

Reply:
3 Levels is the pattern of [Semantic Versioning](https://semver.org/). So the pattern MAJOR.MINOR.PATCH. The version number denotes the importance of the change. Changing the version number from 1000 to 2000 say doesn't give the developer a clue on how big the change was.